web/app/scripts/custom-reports/custom-report-partial.html

```diff
@@ -7,14 +7,20 @@
                 <span>{{ ::ctl.params.occurred_max | localDateTime : ctl.dateFormat }}</span>
             </h2>
             <h3>
-                Incidents
-                <span ng-if="ctl.params.aggregation_boundary"> for {{ ::ctl.report.table_labels[table.tablekey] }}</span>
-                by {{ ::ctl.rowCategoryLabel }} and {{ ::ctl.colCategoryLabel }}
+                {{ ctl.recordType.plural_label }}
+                <span ng-if="ctl.params.aggregation_boundary">
+                  {{ 'COMMON.FOR' | translate }}
+                  {{ ::ctl.report.table_labels[table.tablekey] }}
+                </span>
+                {{ 'COMMON.BY' | translate }}
```

Comment:
I talked to Arianna and the solution we agreed on is that we should change these to be more specific keys, something like `CUSTOM_REPORT.AREA_FOR`, `CUSTOM_REPORT.CATEGORY_BY`, and `CUSTOM_REPORT.CATEGORY_AND`. She said there may still be some issues if the word `by` changes depending on which type of categories we have, but we can work that out with the translators.